mysqld_exporter.go

```diff
@@ -262,6 +264,13 @@ const (
 		  FROM information_schema.schemata
 		  WHERE SCHEMA_NAME NOT IN ('mysql', 'performance_schema', 'information_schema')
 		`
+	queryResponseCheckQuery = `SELECT @@query_response_time_stats`
+	queryResponseTimeQuery = `
+		SELECT
+		    TIME, COUNT, TOTAL
+		  FROM information_schema.query_response_time
+		  WHERE TIME <> 'TOO LONG'
```

Comment:
I think you'd need to special case the TOO LONG and add it to the histogramSum. The client code will take care of the rest.